tests/kokkos/subview_utils.cpp

```diff
@@ -1,376 +1,12 @@
 #include <catch2/catch.hpp>
 
-#include "ekat/kokkos/ekat_subview_utils.hpp"
```

Comment:
I simply split `kokkos_utils_tests.cpp` into separate files: `subview_utils.cpp`, `view_utils.cpp`, and `team_policy_utils.cpp`, for ease of maintenance.